### ISSUE TITLE
Error: ModelCheckpoint(monitor='val_loss') not found in the returned metrics

### ISSUE THREAD

**Issue by Alegzandra** [curiousily/Getting-Things-Done-with-Pytorch#23]
Hi,
When running the notebook 11.multi-label-text-classification-with-bert.ipynb
I encounter the following error...
pytorch_lightning.utilities.exceptions.MisconfigurationException: ModelCheckpoint(monitor='val_loss') not found in the returned metrics: ['train_loss']. HINT: Did you call self.log('val_loss', tensor) in the LightningModule?

Do you know how I could fix this?
